### DISCUSSION TITLE
unit 1 exercise 3 learning rate question

### DISCUSSION THREAD

**Discussion by Nevermetyou65** [Lightning-AI/dl-fundamentals#82] [Exercises] [answered]
I am just a little bit confused with the instruction for this exercise. 
It said that `multiply the bias and weight update by a variable alpha which defaults ....`
Does this mean I should do something like `self.bias += (error * self.alpha)` ??

Thanks

**Comment by rasbt** [comment, marked as answer]
Yes, you are on the right track!

**Reply by Nevermetyou65** [reply]
Thanks a lot, Sebastian. This is really a good course.

**Reply by rasbt** [reply]
Glad to hear :)
